### ISSUE TITLE
Chinese language Support

### ISSUE THREAD

**Issue by BingZi-233** [CloudNetService/CloudNet-v3#661]
**Description of the new feature:**

The WIKI and console support Chinese language files

**Reasons for inclusion into the CloudNet code base:**

Many Chinese server owners and developers are eager to get community support, there is no data available in China. Only a handful of people know about the software, and even fewer use it. This is a great piece of software, and it deserves to be used by more people.

**Planned audience:**

server administrators

**Preview of code and documentation:**

I'm sorry, we don't have this.

**Comment by BingZi-233** [comment]
The code has been provided to request a merge.

**Comment by Score2** [comment]
Whispering to you, I have translated the language files into Chinese, but they were not merged directly into the develop branch, but in another branch under development.

**Comment by Score2** [comment]
However, it would be more perfect if someone could translate the wiki into Chinese as well.

**Comment by BingZi-233** [comment]
Ironically, WIKI is a bit superficial. There are no examples of actual use, and the most common problem I encountered in using the BC was that it didn't connect properly to the child server. I also noticed that Global/Proxy and Proxy groups appeared, and I couldn't tell which template I was using to create. Adding ports manually is not practical because the ports for the child servers are automatically assigned.

**Comment by BingZi-233** [comment]
Also worth mentioning, where is the Git repository for WIKI? I can try to translate.

**Comment by Score2** [comment]
I tried checking the repository, however I found that my old translations seem to have been removed.  This is the pull request I once submitted: https://github.com/CloudNetService/CloudNet-v3/pull/515

**Comment by BingZi-233** [comment]
Yes, they did delete the Chinese files. I've seen it before, and NOW I'm trying to rejoin it.

**Comment by renzwei** [comment]
感谢翻译工作

**Comment by byRoadrunner** [comment]
Hello,
currently we are working on a rework of the website (and also the docs/wiki).
After this is finished (in English) you will be able to translate it on [Crowdin](https://cloudnetservice.crowdin.com/).
